docs/src/components/page-parts/platform-integrations/QuasarIntegrations.vue

```diff
@@ -76,43 +106,113 @@ const platformIntegrationIcons = [
   mdiAndroid
 ]
 
-const integrationOptions = [
+const ecosystemParts = [
+  {
+    about: 'App Extensions that help you design your UI',
+    options: [
+      {
+        label: 'QCalendar',
+        icon: mdiCalendar,
+        path: 'github.com/quasarframework/quasar-ui-qcalendar'
+      },
+      {
+        label: 'QMarkdown',
+        icon: mdiLanguageMarkdown,
+        path: 'github.com/quasarframework/app-extension-qmarkdown'
+      },
+      {
+        label: 'QMediaPlayer',
+        icon: 'ondemand_video',
+        path: 'github.com/quasarframework/app-extension-qmediaplayer'
+      },
+      {
+        label: 'All AEs',
+        icon: 'extension',
+        path: 'quasar.dev/app-extensions/discover',
```

Comment:
When linking an internal page, avoid the domain part, or it won't work when you publish it into different domains